guillotina_mailer/__init__.py

```diff
@@ -19,7 +19,14 @@
         "utility": "guillotina_mailer.utility.MailerUtility",
         "use_html2text": True,
         "domain": None
-    }
+    },
+    "load_utilities": {
```

Comment:
It isn't an async utility that should be run by guillotina. See https://github.com/guillotinaweb/guillotina_mailer/blob/master/guillotina_mailer/utility.py#L124

Reply:
Thanks ! It was looking like an async utility!